authorization_services/topics/hello-world-deploy.adoc

```diff
@@ -66,7 +66,7 @@ This change specifies to the policy enforcer to redirect users to a `/app-authz-
 To build and deploy the application execute the following command:
 
 ```bash
```

Comment:
This won't work as it simply outputs {quickstartRepo_dir} and not the value of the variable. You need to wrap it in:
````
[source,bash,subs="attributes+"]
----
----
````
See https://raw.githubusercontent.com/keycloak/keycloak-documentation/master/securing_apps/topics/oidc/keycloak-gatekeeper.adoc for an example